lib/src/base-styles/_nx-icon.scss

```diff
@@ -5,6 +5,7 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 .nx-icon {
+  color: currentColor;
```

Comment:
This is no longer relevant to this PR

Reply:
> This is no longer relevant to this PR

I thought I'd reverted all those changes: https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/36d87acc73d8e9e53fb4be9bceb1b1204f757cc1